receipt_upload/test/test_dedupe_grand_total.py

```
@@ -0,0 +1,97 @@
+        receipt_id=1,
+        line_id=line_id,
+        word_id=word_id,
+        label=label,
+        reasoning="test",
+        timestamp_added="2026-01-01T00:00:00.000+00:00",
+        validation_status=status,
+    )
+
+
+def _keys(labels):
+    return {(l.line_id, l.word_id) for l in labels}
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)